azkaban-common/src/main/java/azkaban/user/Permission.java

```diff
@@ -194,7 +194,14 @@ public int toFlags() {
     // Users with this permission can upload projects when the property "lockdown.upload.projects"
     // is turned on
     UPLOADPROJECTS(0x0008000),
-    ADMIN(0x8000000);
+    ADMIN(0x8000000),
+    // Permissions for image management APIs.
+    CREATE(0x0000100),
+    GET(0x0000200),
+    UPDATE(0x0000300),
+    DELETE(0x0000400),
+    IMAGE_TYPE_ADD_MEMBER(0x0000500),
```

Comment:
What is this number value for? How is it used?

Reply:
How is it used? How did you get these numbers for new Permission which you have added?

Reply:
These numbers are not used. Added for completeness as these permissions are part of existing code.